### PR TITLE
Route backend

### PR DESCRIPTION
**Pull request by lottiely**
### Added
- Update-fortune method
`Toggles approved item`
- Routed delete, get-by-id, update-fortune, and createItem methods
- Added basic docstrings

### Related
#2 
commit c383604759b91f67a196fc41d33b33b164da9f9f 1b349b5db5074ad05e23d054b932c3b97063a63a

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -62,8 +62,13 @@ def delete_fortune(fortune_id):
         logger.debug("DeleteItem succeeded")
         logger.debug(json.dumps(response, indent=4))
         logger.debug(json.dumps(get_all_fortunes(), indent=4))
+        return response
 
 def createItem(fortune_item):
+    """
+
+    """
```
Comment (ericpak) [inline src/app.py:77]: Nice docstrings

Comment (GiantRobato) [reply src/app.py:77]: 👍

Comment (ericpak) [reply src/app.py:77]: Much Wow :dog:

Comment (lottiely) [reply src/app.py:77]: Thanks. :)

File: src/app.py [hunk 2]
```diff
@@ -89,21 +157,47 @@ def home():
...
+        }
+        pass
 
-    response_obj = {
-        "fortune" : random.choice(fortunes),
-        "status": "success" 
-    }
+    if request.method == "POST":
+        response_obj = createItem(request.json)
+        pass
```
Comment (GiantRobato) [inline src/app.py]: remove this

File: src/app.py [hunk 3]
```diff
@@ -89,21 +157,47 @@ def home():
...
+def fortune(): 
+    response_obj = "Error"
+
+    if request.method == "GET":
+        fortunes = get_all_fortunes()
+        response_obj = {
+            "fortune" : random.choice(fortunes),
+            "status": "success" 
+        }
+        pass
```
Comment (GiantRobato) [inline src/app.py]: remove this

File: src/app.py [hunk 4]
```diff
@@ -89,21 +157,47 @@ def home():
...
+        response_obj = update_fortune(_id)
+        pass
+
+    if request.method == "DELETE":
+        response_obj = delete_fortune(_id)
+        pass
+
+    if request.method == "GET":
+        response_obj = get_by_id(_id)
+        pass
```
Comment (GiantRobato) [inline src/app.py]: remove all pass statements

**Review by GiantRobato — CHANGES_REQUESTED** [review]
remove all pass to pass :D

File: src/app.py (hunk 2 above)
Comment (lottiely) [reply src/app.py]: Thanks! I got rid of them! :3c

**Review by GiantRobato — APPROVED** [review]
Looks Good!

**Pull request merged**